src/v/coproc/script_context.cc

```diff
@@ -168,6 +168,13 @@ ss::future<> script_context::send_request(
 }
 
 bool script_context::is_up_to_date() const {
+    if (_routes.empty()) {
+        /// Since this method is only used for unit tests, its never desired to
```

Comment:
it's customary to name such methods `testing_only_xxx`

Reply:
This method is safe to be used outside of tests as long as the caller understands how it should behave when theres no registered routes